### PR TITLE
Add rp-rs to list of chats

### PR DESCRIPTION
**Pull request by robyoung**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ In 2018 the Rust community created an embedded working group to help drive adopt
 - [#esp-rs:matrix.org] - For discussion of using Embedded Rust on Xtensa devices
 - [#nrf-rs:matrix.org] - For discussion of using Embedded Rust on Nordic Semiconductor devices
 - [#probe-rs:matrix.org] - For discussion of the Probe-rs debugging toolkit
+- [#pr-rs:matrix.org] - For discussion of using Embedded Rust on RP2040 based devices
```
Comment (eldruin) [inline README.md:89]:
```suggestion
- [#rp-rs:matrix.org] - For discussion of using Embedded Rust on RP2040 based devices
```

File: README.md [hunk 2]
```diff
@@ -101,6 +102,7 @@ In 2018 the Rust community created an embedded working group to help drive adopt
 [`embedded-graphics`]: https://crates.io/crates/embedded-graphics
 [#stm32-rs:matrix.org]: https://matrix.to/#/#stm32-rs:matrix.org
 [#avr-rust:gitter.im]: https://matrix.to/#/#avr-rust_Lobby:gitter.im
+[#pr-rs:matrix.org]: https://matrix.to/#/#rp-rs:matrix.org
```
Comment (eldruin) [inline README.md:105]:
```suggestion
[#rp-rs:matrix.org]: https://matrix.to/#/#rp-rs:matrix.org
```

**Review by eldruin — COMMENTED** [review]
Other than the typo, looks good to me, thanks!

**Review by eldruin — APPROVED** [review]
bors r+

File: README.md (hunk 1 above)
Comment (robyoung) [reply README.md]: Oh dear, sorry, and thanks.

**Pull request merged**